Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,25 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
+//We want to log QTY, ITEM, and TOTAL once at the beginning of the receipt
+//Then we want to log each item in the order array, and align the columns
+//Finally, we want to log the total price of the order
+function printReceipt(order){
+  let total=0
+  console.log("QTY     ITEM                  TOTAL")
+  console.log("--------------------------------");
+
+  order.forEach(({itemName, quantity, unitPricePence})=>{
+    let totalItem=quantity*unitPricePence/100
+    total+=totalItem
+    console.log(
+      `${quantity.toString().padEnd(7)} ${itemName.padEnd(21)} £${totalItem.toFixed(2)}`
+    );
+  })
+  console.log(`Total: ${total.toFixed(2)}`)
```

Comment:
1. The readme file shows a space between the last item and the final total. Can you make this small change?
2. The readme does not show the item.length being output. Can you update your code to reflect the expected result from the readme?